fireant/tests/mock_dataframes.py

```diff
@@ -19,13 +19,13 @@ def rollup(dataframe, levels):
     return dataframe.append(roll.set_index(rolled_levels, append=True)).sort_index()
 
 
-cont_dim = {'label': 'Cont'}
-datetime_dim = {'label': 'Date'}
-uni_dim = {'label': 'Uni', 'display_field': 'uni_label'}
-cat1_dim = {'label': 'Cat1', 'display_options': {'a': 'A', 'b': 'B'}}
-cat2_dim = {'label': 'Cat2', 'display_options': {'y': 'Y', 'z': 'Z'}}
-cat1_rollup_dim = {'label': 'Cat1', 'display_options': {'a': 'A', 'b': 'B', Totals.key: Totals.label}}
-cat2_rollup_dim = {'label': 'Cat2', 'display_options': {'y': 'Y', 'z': 'Z', Totals.key: Totals.label}}
+cont_dim = {'axis': 0, 'label': 'Cont'}
```

Comment:
@twheys The 0 will always be present. Non-reference series have axis indexes as counts starting from 0. That was always how we treated axis indexes.